go.mod

```diff
@@ -82,6 +84,7 @@ require (
 	github.com/rs/cors v1.6.0
 	github.com/ryanuber/go-glob v1.0.0 // indirect
 	github.com/satori/go.uuid v1.2.0 // indirect
+	github.com/sirupsen/logrus v1.4.2
```

Comment:
This seems unrelated.

Reply:
Again auto-added by `go mod` due to `go mod tidy`

Reply:
ah, ok :)